install/kubernetes/helm/istio/charts/grafana/dashboards/pilot-dashboard.json

```diff
@@ -1696,18 +1490,20 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "sum(pilot_xds_eds_instances{job=\"pilot\"}) by (cluster)",
+          "expr": "count(sum(pilot_xds_eds_instances{job=\"pilot\", cluster=~\".+\\\\|.+\"}) by (cluster) < 1)",
```

Comment:
Should we make this a table? Right now it tells me I have 12 clusters with no endpoints but I don't know which ones

Reply:
i had this as a table to start, but then couldn't convince myself which way was better. i'm happy to have it be a table if that would be more useful.

Reply:
The reason I was leaning towards a table is that it doesn't seem actionable right now. You see you have some clusters without endpoints and then ...? 

I am curious what made you not want a table? It would be one of our only tables which is a bit weird